src/ui/sass/modules/_Icon.scss

```diff
@@ -160,3 +160,15 @@
     }
   }
 }
+
+.Icon--magnifying_glass svg
+{
```

Comment:
This is just a static magnifying glass without the animation from yext to it, but will replace the original animated magnifying glass icon if it's a transition from loading state to non-loading. since this not used in a children div of the loading indicator, so by scoping it, you mean renaming it to something related to loading indicator?

Reply:
updated